src/js/tabs/pid_tuning.js

```diff
@@ -451,8 +451,20 @@ TABS.pid_tuning.initialize = function (callback) {
             });
             ffInterpolateCheck.change();
 
+            // Vbat Sag Compensation
+            const vbatSagCompensationCheck = $('input[id="vbatSagCompensation"]');
+
+            vbatSagCompensationCheck.prop('checked', ADVANCED_TUNING.vbat_sag_compensation !== 0);
+            $('input[name="vbatSagValue"]').val(ADVANCED_TUNING.vbat_sag_compensation > 0 ? ADVANCED_TUNING.vbat_sag_compensation : 100);
```

Comment:
When it is zero the value is 100? Maybe this is the default value or something similar?

Reply:
@McGiverGim yes, 100 is our default value when is enabled this feature recomended by @ctzsnooze